lib/commands/arm/group/group.deployment._js

```
@@ -119,6 +118,9 @@ exports.init = function (cli) {
             displayDeployment(data, resourceGroup, true, log);
           }
         });
+        if (deployment.properties && utils.ignoreCaseEquals(deployment.properties.provisioningState, 'Failed')) {
+          throw new Error($('Deployment provisioning state was not successful.'));
```

why throw an error instead of log.error ?